doc/source/user_guide_session/index.rst

```diff
@@ -21,14 +21,13 @@ to a remote Mechanical session.
 Overview
 --------
 
-The :func:`launch_mechanical() <ansys.mechanical.core.launch_mechanical>` method
-creates an instance of the :class:`Mechanical <ansys.mechanical.core.mechanical.Mechanical>`
+The `launch_mechanical() <https://mechanical.docs.pyansys.com/version/stable/api/ansys/mechanical/core/mechanical/index.html#mechanical.launch_mechanical>`_ method
```

Comment:
What happens if we have some change in dev version of launch.mechanical ? Will it be redirected to stable version in dev docs ?

Reply:
Yes I believe so

Reply:
Should we change the links to version/dev then? but then we'll have that issue with the stable docs